src/modules/mtev_kafka.h

```diff
@@ -115,6 +126,20 @@ typedef struct mtev_rd_kafka_message {
   void (*free_fn)(struct mtev_rd_kafka_message *m);
 } mtev_rd_kafka_message_t;
 
+typedef struct mtev_kafka_connection_info {
+  mtev_kafka_connection_type_e connection_type;
+  uuid_t id;
+  char *host;
+  int32_t port;
+  size_t topic_count;
+  char **topics;
+} mtev_kafka_connection_info_t;
+
+typedef struct mtev_kafka_connection_list {
+  mtev_kafka_connection_info_t *connections;
+  size_t count;
+} mtev_kafka_connection_list_t;
+
```

Comment:
I would consider making these typedefs opaque and providing accessors to get things the callers might need.  The exposure of these makes people think they could modify them and also will break the ABI if we alter them at all.  If they are opaque pointers we can fix the struct (add/remove/reorder elements) without break a compiled client.